src/main/java/cz/upce/fei/inptp/zz/entity/School.java

```diff
@@ -18,23 +14,101 @@ public School() {
         teachers = new ArrayList<>();
         students = new ArrayList<>();
     }
+
+    public boolean checkIsStudentAtSchool(Student student) {
+        for (Student st : students) {
+            if (st.equals(student)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public boolean checkIsCourseAtSchool(Course course) {
+        for (Course course1 : courses) {
+            if (course1.equals(course)) {
```

Comment:
And there is course and course1 - again not really useful names and inconsistent with previous method.